ext_emconf.php

```diff
@@ -21,8 +21,8 @@
     'category' => 'be',
     'constraints' => [
         'depends' => [
-            'typo3' => '10.4.0-10.4.99',
-            'dashboard' => '10.4.0-10.4.99',
+            'typo3' => '10.4.0-11.5.99',
```

Comment:
I get the point here, but it's basically only true is no breaking changes doesn't come into the core. It's planned like that, but we never know. 

So considering if we should keep it at 11.0.99 for  now?

Reply:
I want to avoid that every two months there must be another release to be installable with 11.1, 11.2, 11.3, etc. If there will be a breaking change (which will be avoided by the core team) then it should be considered as a bug.
Also Benni marks his extensions with 11.5.99, e.g.: https://extensions.typo3.org/extension/geocoding/